apps/pages/src/utils/newPageContentObject.js

```diff
@@ -3,6 +3,6 @@ export function newPageContentObject() {
     css: "",
     html: "",
     javascript: "",
-    dependencies: { css: [], javascript: [] },
+    dependencies: { css: [], javascript: [], jsDefer: true },
```

Comment:
defer should be per dependency. not all or nothing.

other option: also support the 'original' full text mode.

Reply:
This is now fixed. defer can be set for each dependency